src/lib/prisma.ts

```diff
@@ -0,0 +1,19 @@
+// ts-ignore 7017 is used to ignore the error that the global object is not
```

Comment:
This file may look complicated but it is just boilerplate code to make sure that the prisma client is efficiently instantiated. It is copy/paste from set up guides.